site/profile/templates/cvmfs/soft.mugqic.conf.epp

```diff
@@ -0,0 +1,4 @@
+CVMFS_REPOSITORIES=soft.mugqic
+CVMFS_ALIEN_CACHE="/scratch/cvmfs_alien_cache/soft.mugqic"
+CVMFS_QUOTA_LIMIT='-1'
+CVMFS_SHARED_CACHE='no'
```

Comment:
no need to add the config with soft.mugqic. I would only be added at the end of the `CVMFS_REPOSITORIES` list in cvmfs/default.local